apps/hosts/views.py

```diff
@@ -152,7 +152,7 @@ def challenge_host_list(request, challenge_host_team_pk):
 @throttle_classes([UserRateThrottle])
 @permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
 @authentication_classes((ExpiringTokenAuthentication,))
-def challenge_host_detail(request, challenge_host_team_pk, pk):
+def challenge_host_delete(request, challenge_host_team_pk, challenge_host_pk):
```

Comment:
I think we should not change the name of this function since it takes all the requests `GET`, `PUT`, `PATCH` and `DELETE`.